rag_app_sample_code/A_POC_app/databricks_docs_example/00_config.py

```diff
@@ -95,11 +95,11 @@
 # Names of the output Delta Tables tables & Vector Search index
 destination_tables_config = {
     # Staging table with the raw files & metadata
-    "raw_files_table_name": f"{UC_CATALOG}.{UC_SCHEMA}.{RAG_APP_NAME}_poc_raw_files_bronze",
```

Comment:
I believe this would be simpler if you we did `{UC_CATALOG}.{UC_SCHEMA}.{RAG_APP_NAME}_poc_raw_files_bronze` rather than `` around each part

Reply:
If we don't put the `` around each part, then catalog names containing a '-' will not work for any SQL command.
Adding the `` fixes the issue for SQL, but breaks the vector store API calls, so we need to either align the vector store APIs to accept `` or use the approach I propose in this PR to add `` to all table names and then remove them when calling the vector store APIs.